packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -720,7 +782,8 @@ export default function CdcDashboard({ configUrl = '', config: configObj = undef
             </div>
           )
         })}
-        {config.filterBehavior === FilterBehavior.Apply && <button onClick={applyFilters}>GO!</button>}
+
+        {!isAutoLoadRow && <button onClick={applyFilters}>GO!</button>}
```

Comment:
The default for already saved configs will be not not have the FilterBeahvior.Apply set, however this will still show the go button in that case. Need to rework logic to hide go button when FilterBehavior is unset.

Reply:
@mpallansch I added an isLegacyFilter variable which may resolve this? can you review that and let me know? I added a comment next to the variable